core/vcpu.c

```diff
@@ -78,7 +78,6 @@ static void vcpu_init(struct vcpu_t *vcpu);
 static void vcpu_prepare(struct vcpu_t *vcpu);
 static void vcpu_init_emulator(struct vcpu_t *vcpu);
 
-static void vmread_cr(struct vcpu_t *vcpu);
```

Comment:
Is this for clean-up? If so, we can do it together in future.

Reply:
Yes, cleanup.